wsd/COOLWSD.cpp

```diff
@@ -2547,6 +2548,15 @@ void COOLWSD::innerInitialize(Application& self)
     setenv("DEEPL_API_URL", apiURL.c_str(), 1);
     setenv("DEEPL_AUTH_KEY", authKey.c_str(), 1);
 
+#if !MOBILEAPP
+    const std::string helpUrl = getConfigValue<std::string>(conf, "help_url", HELP_URL);
+    setenv("LOK_HELP_URL", helpUrl.c_str(), 1);
```

Comment:
Is there already logic that read LOK_HELP_URL envar and uses it? Is that in a separate PR?

Reply:
yes, see https://gerrit.libreoffice.org/c/core/+/151739